helpers/config.py

```diff
@@ -765,16 +775,50 @@ def __questions_aws(self):
             'Do you want to use AWS S3 storage?',
             default=self.__dict['use_aws']
         )
+
+        MAXIMUM_AWS_CREDENTIAL_ATTEMPTS = 2
+        aws_credential_attempts = 0
         if self.__dict['use_aws'] is True:
-            self.__dict['aws_access_key'] = CLI.colored_input(
-                'AWS Access Key', CLI.COLOR_QUESTION,
-                self.__dict['aws_access_key'])
-            self.__dict['aws_secret_key'] = CLI.colored_input(
-                'AWS Secret Key', CLI.COLOR_QUESTION,
-                self.__dict['aws_secret_key'])
-            self.__dict['aws_bucket_name'] = CLI.colored_input(
-                'AWS Bucket name', CLI.COLOR_QUESTION,
-                self.__dict['aws_bucket_name'])
+            while (
+                not self.__dict.get('aws_credentials_valid')
+            ) and (aws_credential_attempts <= MAXIMUM_AWS_CREDENTIAL_ATTEMPTS):
+                self.__dict['aws_access_key'] = CLI.colored_input(
+                    'AWS Access Key', CLI.COLOR_QUESTION,
+                    self.__dict['aws_access_key'])
+                self.__dict['aws_secret_key'] = CLI.colored_input(
+                    'AWS Secret Key', CLI.COLOR_QUESTION,
+                    self.__dict['aws_secret_key'])
+                self.__dict['aws_bucket_name'] = CLI.colored_input(
+                    'AWS Bucket name', CLI.COLOR_QUESTION,
+                    self.__dict['aws_bucket_name'])
+
+                aws_credential_attempts += 1
+                self.validate_aws_credentials()
+                attempts_remaining = (
+                    MAXIMUM_AWS_CREDENTIAL_ATTEMPTS - aws_credential_attempts
+                )
+                if (not self.__dict.get('aws_credentials_valid')) and (
+                    attempts_remaining >= 0
+                ):
+                    CLI.colored_print(
+                        f'Attempts remaining: {attempts_remaining}',
+                        CLI.COLOR_INFO,
+                    )
+            else:
+                if not self.__dict.get('aws_credentials_valid'):
+                    self.__dict['aws_skip_validation'] = CLI.yes_no_question(
```

Comment:
It should bypass the validation next time users run the setup. 
Maybe the logic should be more like other questions (e.g. Postgres settings, customize ports or modify `/etc/hosts`)
For example: 
- Enter AWS Credentials
- Ask if user want to validate
- if yes, enter the loop 
- if no, keep going to next question

What do you think?

Reply:
> It should bypass the validation next time users run the setup.

I guess the user could be changing their S3 credentials in the next setup? I don't know if that's a likely scenario.

> What do you think?

I like that 👍 it gives the user a bit more control that way.